pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -58,30 +56,29 @@ func ValidateKubeSchedulerConfiguration(cc *config.KubeSchedulerConfiguration) f
 			}
 			existingProfiles[profile.SchedulerName] = i
 		}
-		allErrs = append(allErrs, fromAggregate(utilerrors.Flatten(utilerrors.NewAggregate(errs)))...)
-		allErrs = append(allErrs, validateCommonQueueSort(profilesPath, cc.Profiles)...)
+		errs = append(errs, validateCommonQueueSort(profilesPath, cc.Profiles).ToAggregate())
```

Comment:
Use `...`

Reply:
You can still do this one by changing the return type of `validateCommonQueueSort` to `[]error`

Reply:
Done